lib/openstax_kitchen.rb

```diff
@@ -51,7 +51,7 @@ def require_all(relative_folder, file_matcher='**/*.rb')
 
 require_all('kitchen/directions')
 
-I18n.load_path << file_glob('/locales/*.yml')
+I18n.backend.load_translations(file_glob('/locales/*.yml'))
```

Comment:
Same as above `load_path` comment, `load_path` impacts all backends, so when I made a new temporary backend, it also got all of the kitchen locales loaded onto it.  By calling `load_translations` instead, we avoid that.